src/runtime/runtime.h

```diff
@@ -735,6 +735,8 @@ namespace internal {
   F(Float32x4WithY, 2, 1)                      \
   F(Float32x4WithZ, 2, 1)                      \
   F(Float32x4WithW, 2, 1)                      \
```

Comment:
The Float32x4WithX/Y/Z/W should be replaced by Float32x4ReplaceLane. Ditto for Float64x2 and Int32x4.